project/MimaExcludes.scala

```diff
@@ -319,6 +319,68 @@ object MimaExcludes {
         ProblemFilters.exclude[IncompatibleResultTypeProblem]("org.apache.spark.ml.classification.LogisticRegressionSummary.predictions"),
         ProblemFilters.exclude[MissingMethodProblem]("org.apache.spark.ml.classification.LogisticRegressionSummary.predictions")
       ) ++ Seq(
+        // [SPARK-13660][SQL][Tests] ContinuousQuerySuite floods the logs with garbage
```

Comment:
can you undo all the mima changes? there was a problem with the build and we fixed that.